### ISSUE TITLE
Upgrading to Laravel 6.0 Translator::trans does not exist error. 

### ISSUE THREAD

**Issue by juliancc** [kristijanhusak/laravel-form-builder#557]
```
Facade\Ignition\Exceptions\ViewException
Method Illuminate\Translation\Translator::trans does not exist.
```

inside of RulesParser.php
```php   
    protected function getTitle($rule, $params = array())
    {
        $params['attribute'] = $this->field->getOption('label');

        return $this->formHelper->getTranslator()->trans('validation.' . $rule, $params);
    }
```

change ->trans to ->get:

```php    
    protected function getTitle($rule, $params = array())
    {
        $params['attribute'] = $this->field->getOption('label');

        return $this->formHelper->getTranslator()->get('validation.' . $rule, $params);
    }

**Comment by devyk** [comment]
Temporary solution without forking. Until the issue will be resolved.

```
        Translator::macro('trans', function(...$args) {
            return call_user_func_array('__', $args);
        });
```

**Comment by rudiedirkx** [comment]
Was fixed in 3de891f43a01ae1fa4df7eb9d2e6cc93f5ebfba9
